downstream/assemblies/hub/assembly-synclists.adoc

```diff
@@ -1,6 +1,15 @@
 [id="assembly-synclists"]
 = Managing Ansible Content Collection synclists in {HubName}
 
+[IMPORTANT]
+====
+As of the 2.4 release you can still synchronize content, but synclists are deprecated, and will be removed in a future version.
+
+From Ansible Automation Platform 2.4 a private automation hub administrator can go to the rh-certified remote and upload a manually-created requirements file. 
+
+Remotes are configurations that allow you to synchronize content to your custom repositories from an external collection source.
+====
+
 You can use {HubNameMain} to distribute the relevant Red Hat Certified collections content to your users by creating synclists.
```

Comment:
If you make the suggested change above, change this to "...by creating synclists or a requirements file. For more information about using requirements files, see [Install multiple collections with a requirements file](https://docs.ansible.com/ansible/latest/galaxy/user_guide.html#install-multiple-collections-with-a-requirements-file) in the Galaxy User Guide."